README.md

```diff
@@ -8,47 +10,39 @@
 [actions-badge]: https://github.com/eigen-trust/protocol/actions/workflows/test.yml/badge.svg
 [actions-url]: https://github.com/eigen-trust/protocol/actions?query=branch%3Amaster
 
-A library for managing trust in a distributed network with zero-knowledge features.
-
-## Main characteristics:
-
-**Self-policing** - the shared ethics of the user population is defined and enforced by the peers themselves and not by some central authority.
+A Rust and Halo2-based library designed to manage trust in distributed networks with zero-knowledge proofs, guided by the framework established in the original [EigenTrust paper](https://nlp.stanford.edu/pubs/eigentrust.pdf). Its primary characteristics are:
 
-**Minimal** - computation, infrastructure, storage, and message complexity are reduced to a minimum.
+- **Self-policing**: The shared ethics of the user population is defined and enforced by the peers themselves and not by some central authority.
 
-**Incorruptible** - Reputation should be obtained by consistent good behavior through several transactions. This is enforced for all users, so no one can cheat the system and obtain a higher reputation. It is also resistant to malicious collectives.
+- **Minimal**: computation, infrastructure, storage, and message complexity are reduced to a minimum.
 
-## Development Guidelines
+- **Incorruptible**: Reputation should be obtained by consistent good behavior through several transactions. This is enforced for all users, so no one can cheat the system and obtain a higher reputation. It is also resistant to malicious collectives.
 
-The following document explains the development process and our values:
+## Deprecation Notice
 
-[Development Process](https://hackmd.io/MzCV5EGyTo-aNIRUV0PnUQ)
+Effective November 2023, this project has been deprecated and is no longer under active development.
```

Comment:
Same here.
How about `Effective November 2023, this project is no longer under active development`?